test/functional/rpc_packages.py

```diff
@@ -158,156 +148,116 @@ def test_chain(self):
 
         # Ensure tx1 and tx2 are valid by themselves
-        assert node.testmempoolaccept([signedtx1["hex"]])[0]["allowed"]
-        assert node.testmempoolaccept([signedtx2["hex"]])[0]["allowed"]
+        assert node.testmempoolaccept([tx1["hex"]])[0]["allowed"]
+        assert node.testmempoolaccept([tx2["hex"]])[0]["allowed"]
 
         self.log.info("Test duplicate transactions in the same package")
-        testres = node.testmempoolaccept([signedtx1["hex"], signedtx1["hex"]])
+        testres = node.testmempoolaccept([tx1["hex"], tx1["hex"]])
         assert_equal(testres, [
-            {"txid": tx1.rehash(), "wtxid": tx1.getwtxid(), "package-error": "conflict-in-package"},
-            {"txid": tx1.rehash(), "wtxid": tx1.getwtxid(), "package-error": "conflict-in-package"}
+            {"txid": tx1["txid"], "wtxid": tx1["wtxid"], "package-error": "conflict-in-package"},
+            {"txid": tx1["txid"], "wtxid": tx1["wtxid"], "package-error": "conflict-in-package"}
         ])
 
         self.log.info("Test conflicting transactions in the same package")
-        testres = node.testmempoolaccept([signedtx1["hex"], signedtx2["hex"]])
+        testres = node.testmempoolaccept([tx1["hex"], tx2["hex"]])
         assert_equal(testres, [
-            {"txid": tx1.rehash(), "wtxid": tx1.getwtxid(), "package-error": "conflict-in-package"},
-            {"txid": tx2.rehash(), "wtxid": tx2.getwtxid(), "package-error": "conflict-in-package"}
+            {"txid": tx1["txid"], "wtxid": tx1["wtxid"], "package-error": "conflict-in-package"},
+            {"txid": tx2["txid"], "wtxid": tx2["wtxid"], "package-error": "conflict-in-package"}
         ])
 
     def test_rbf(self):
         node = self.nodes[0]
-        coin = self.coins.pop()
-        inputs = [{"txid": coin["txid"], "vout": 0, "sequence": MAX_BIP125_RBF_SEQUENCE}]
+
+        coin = self.wallet.get_utxo()
         fee = Decimal('0.00125000')
-        output = {node.get_deterministic_priv_key().address: 50 - fee}
-        raw_replaceable_tx = node.createrawtransaction(inputs, output)
-        signed_replaceable_tx = node.signrawtransactionwithkey(hexstring=raw_replaceable_tx, privkeys=self.privkeys)
-        testres_replaceable = node.testmempoolaccept([signed_replaceable_tx["hex"]])
-        replaceable_tx = tx_from_hex(signed_replaceable_tx["hex"])
+        replaceable_tx = self.wallet.create_self_transfer(utxo_to_spend=coin, sequence=MAX_BIP125_RBF_SEQUENCE, fee = fee)
+        testres_replaceable = node.testmempoolaccept([replaceable_tx["hex"]])
         assert_equal(testres_replaceable, [
-            {"txid": replaceable_tx.rehash(), "wtxid": replaceable_tx.getwtxid(),
-            "allowed": True, "vsize": replaceable_tx.get_vsize(), "fees": { "base": fee }}
+            {"txid": replaceable_tx["txid"], "wtxid": replaceable_tx["wtxid"],
+            "allowed": True, "vsize": replaceable_tx['tx'].get_vsize(), "fees": { "base": fee }}
```

Comment:
Prefer double quotes for consistency

Reply:
Done in https://github.com/bitcoin/bitcoin/commit/17cad448516a6906ff637593ab57df332fade5d2.